### DISCUSSION TITLE
Minor issue: download update indicator

### DISCUSSION THREAD

**Discussion by kds112** [devemux86/cruiser#435] [Q&A] [answered]
I have downloaded 10 OpenAndroMaps maps, the Elevate theme and the mapsforge world map. Today 7 updates were indicated on the offline map tile. The update process worked properly. After finishing all updates there was still the indication of one available update but the update folder is empty. Closing and reopen the menu does not help. One available update is still indicated on the tile. Closing and restarting the app solved this small bug.
Maybe you can have a look to it but it is just a minor issue.

**Comment by devemux86** [comment]
@kds112 thanks for the report.

The updates were just for the OpenAndroMaps maps (7 / 10)?
Not also update for the Elevate theme?

**Reply by kds112** [reply]
Exactly, just map updates.

**Reply by r11d11** [reply]
I first thought I have the same issue but I only have to switch from Mapsforge to OpenAndroMaps to see the update. It is a little bit confusing when you see offline maps(1) and poi (1) and you see no update because the update is for OpenAndroMaps but you have to switch to Maps Forge to see the update.

**Reply by kds112** [reply]
But I had no mapsforge updates, just OAM maps updates.

**Comment by devemux86** [comment]
Too many automations can often be complicated, especially with 3 download servers working differently.

We should try to simplify things.

**Reply by kds112** [reply]
Yes, please 👍 😇

**Comment by devemux86** [comment, marked as answer]
Resolved in [Cruiser 3.0.39](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
